### PR TITLE
SC-6566: i18n some more missing keys

### PR DESCRIPTION
**Pull request by Tratori**
https://ticketsystem.schul-cloud.org/browse/SC-6566

### CODE REVIEW

File: locales/de.json [hunk 1]
```diff
@@ -1345,6 +1347,24 @@
 			"updatedAt": "Änderungsdatum",
 			"write": "Schreiben"
 		},
+		"overview" : {
+			"button" : {
+				"openFiles" : "Dateien öffnen"
+			},
+			"headline":{
+				"myCourseFiles" : "Meine Kurs-Dateien",
+				"myPersonalFiles" : "Meine persönlichen Dateien",
```
Comment (ABialoskorski) [inline locales/de.json:1356]: duplicate

File: locales/de.json [hunk 2]
```diff
@@ -1345,6 +1347,24 @@
...
+				"mySharedFiles" : "Mit mir geteilte Dateien",
```
Comment (ABialoskorski) [inline locales/de.json:1357]: duplicate

**Review by ABialoskorski — APPROVED** [review]
delete duplicates, I have found two, mocha test did not

**Comment by Tratori** [comment]
> delete duplicates, I have found two, mocha test did not

Mocha Test doesnt look for duplicates between two different "types" of keys, here *.label.* and *.headline.*   ...
I think this is kind of a design flaw with our naming schema, as it leads to lots of duplicates

**Review by ABialoskorski — CHANGES_REQUESTED** [review]
as the requirements changed, please add english translations also :)

**Review by ABialoskorski — APPROVED** [review]

**Review by dominikbruechner — APPROVED** [review]

**Pull request merged**